Start-C4BNexusSteup.ps1

```diff
@@ -995,7 +995,7 @@ if ($LASTEXITCODE -eq 0) {
 else {
     Write-Warning "Microsoft Edge install was not succesful."
     Write-Host "Installing Google Chrome as an alternative."
-    choco install googlechrome -y
+    choco install googlechrome -y --source="'https://chocolatey.org/api/v2/'"
```

Comment:
Same as above

Reply:
Good point! I'll address this in the next commit.